packages/page/translations/admin.de.json

```
@@ -1,4 +1,7 @@
 {
+    "sulu_page.webspaces": "Webspaces",
+    "sulu_page.no_navigation": "keiner Navigation",
+    "sulu_page.all_navigations": "allen Navigationen",
     "sulu_page.pages": "Seite",
     "sulu_page.page": "Seite",
     "sulu_page.no_page_selected": "Kein Seite ausgewählt",
```

```suggestion
    "sulu_page.no_page_selected": "Keine Seite ausgewählt",
```

We should add a TODO to the roadmap issue to go over all translations, I guess when we copied the article bundle to create the page/snippet bundle, we did never fix the translations 😅

---

done